[CAMEL-14581] Remove sub-menus for documentation and Reorganize Documentation menu (#216)
apache/camel-website: content/documentations/_index.md

---

> title: Documentations for Apache Camel

Reviewer: I'd go with:
```suggestion
title: Apache Camel documentation
```

and place this file in `content/documentation`.

Author: I have added all the suggestions related to rephrasing of content.

---

> 

Reviewer: I think we need an introduction here on how to find the right documentation and how to contribute to the documentation.

Author: I hope the changes are better now!

## User Manual

> The User Manual is a comprehensive guide meant to answer all your questions - from how to begin contributing to Apache Camel, to architecture and integration patterns.

Reviewer: Perhaps something like.

```suggestion
The [User Manual](/manual/latest) is a comprehensive guide meant to help you with the key concepts of Apache Camel and software integration, from how to [begin contributing](/manual/latest/contributing.html) to Apache Camel, to [architecture](/manual/latest/architecture.html) or [integration patterns](/components/latest/eips/enterprise-integration-patterns.html).
```

 - [How to Get Started](/manual/latest/getting-started.html)
 - [Enterprise Integration Patterns](/manual/latest/enterprise-integration-patterns.html)
 - [Camel 2.x to 3.0 Migration Guide](/manual/latest/camel-3-migration-guide.html)
 - [Camel 3.x Upgrade Guide](/manual/latest/camel-3x-upgrade-guide.html)

>  - [FULL USER MANUAL](/manual/latest/)

Reviewer: I'd go with:

```suggestion
 - [Complete User manual](/manual/latest/)
```

> ## Project Documentation

Reviewer: Perhaps:

```suggestion
## Camel sub-projects documentation
```

> Apache Camel encompasses several projects which offer many services to the user. Check out the manuals and documentation for these projects here :

Reviewer: ```suggestion
Apache Camel encompasses several sub-projects for additional platforms or ways to run Camel. Check out the individual sub-project manuals here:
```

- [Camel-K](/camel-k/latest/)
- [Camel Quarkus](/camel-quarkus/latest/)
- [Camel Kafka Connector](/camel-kafka-connector/latest/)

> - [JavaDocs documentation](https://www.javadoc.io/doc/org.apache.camel/camel-api/latest/index.html) for Apache's APIs for Camel Core, Camel Catalog, Camel Cloud, Camel Cluster Support and Camel SPI etc.

Reviewer: For text I'd go with something like:

```suggestion
- [API documentation](https://www.javadoc.io/doc/org.apache.camel/camel-api/latest/index.html)
```

Though this relates more to the Camel itself not to the sub-projects, so it needs be with in the user manual section. Perhaps that section needs to be renamed also as it won't be containing just the user manual.

Author: For this, should I move the API documentation to User Manual (and rename it to something else)? Or let it stay under sub-projects for now?

Reviewer: The API documentation is for Camel not for the sub-projects, that's why I would not put the API documentation under sub-projects.

## Components

> [Component Reference Guide](/components/latest/) provides a detailed list of Camel components, their utilities and details about which versions support each component.

Reviewer: Perhaps:

```suggestion
[Component Reference](/components/latest/) is auto generated from the code to reflect the configuration options and functionality available in a particular Camel version.
```
